communication/moderators.md

```diff
@@ -50,22 +88,48 @@ See our [moderation guidelines](./moderating.md) for policies and recommendation
 - Jorge Castro (@castrojo) - ET
 - Joe Beda - (@joebeda) - PT
 - Jaice Singer DuMars - (@jdumars) - PT
+- Open
```

Comment:
Do we have language anywhere about how we should be removing YouTube comments that break code of conduct and that admins trim videos, publish them, set up playlists, etc?

Reply:
Let’s add one more open

Reply:
Added an open and added Bob Killen as a pro tempore to reflect his communications manager status in youtube, which wasn't listed in moderators.md. I added 2 bullets in the youtube section, one for the comments, and one for making sure we keep youtube realtime chat off. 

The automated youtube system is similar to discourse, it can detect bad behavior and put it in a queue - I've added an explicit item for the moderators to check that queue. 

The editing and youtube playlist stuff isn't mentioned anywhere, I'll take an item to put that in the youtube.md.